source/reference/api/reverse.txt

```diff
@@ -40,8 +40,14 @@ To use the ``reverse`` endpoint:
 - You must have started your original sync operation with the
   ``reversible`` and ``enableUserWriteBlocking`` options set to
   ``true``.
-
 - ``mongosync`` must be in the ``COMMITTED`` state.
+- :ref:`Unique indexes <index-type-unique>` must be formatted properly.
+  Format indexes with the :dbcommand:`collMod` command. See `this example
+  <https://www.mongodb.com/docs/v6.0/reference/command/collMod/#convert-an-existing-index-to-a-unique-index>`__
+  for more details.
+- Source and destination clusters must be MongoDB 6.0 or later. If a
+  cluster has been upgraded from MongoDB 4.2 or earlier, an initial sync
```

Comment:
@ldennis What is an initial sync in this context? Is it a forward sync from the legacy 4.2 cluster to the other cluster before reversing? 

What happens if both clusters were upgraded for 4.2?

Reply:
Initial sync means https://www.mongodb.com/docs/manual/tutorial/resync-replica-set-member/#resync-a-member-of-a-replica-set

So they will have to resync a mongod, I think we should link that.

Reply:
They need to initial sync _all_ of their nodes.